docs/csharp/programming-guide/main-and-command-args/index.md

```diff
@@ -41,39 +41,30 @@ translation.priority.ht:
-  
-## In This Section  
-  
--   [Command-Line Arguments](../../../csharp/programming-guide/main-and-command-args/command-line-arguments.md)  
-  
--   [How to: Display Command Line Arguments](../../../csharp/programming-guide/main-and-command-args/how-to-display-command-line-arguments.md)  
-  
--   [How to: Access Command-Line Arguments Using foreach](../../../csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach.md)  
-  
--   [Main() Return Values](../../../csharp/programming-guide/main-and-command-args/main-return-values.md)  
-  
-## C# Language Specification  
- [!INCLUDE[CSharplangspec](~/includes/csharplangspec-md.md)]  
-  
-## See Also  
- [Command-line Building With csc.exe](../../../csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md)   
- [C# Programming Guide](../../../csharp/programming-guide/index.md)   
- [Methods](../../../csharp/programming-guide/classes-and-structs/methods.md)   
- [Inside a C# Program](../../../csharp/programming-guide/inside-a-program/index.md)   
- [\<paveover>C# Sample Applications](http://msdn.microsoft.com/en-us/9a9d7aaa-51d3-4224-b564-95409b0f3e15)
+# Main() and command-line arguments (C# Programming Guide)
+The `Main` method is the entry point of a C# console application or windows application. (Libraries and services do not require a `Main` method as an entry point.). When the application is started, the `Main` method is the first method that is invoked.
+
+ There can only be one entry point in a C# program. If you have more than one class that has a `Main` method, you must compile your program with the **/main** compiler option to specify which `Main` method to use as the entry point. For more information, see [/main (C# Compiler Options)](../../../csharp/language-reference/compiler-options/main-compiler-option.md).
+
+ [!code-cs[csProgGuideMain#17](../../../csharp/programming-guide/inside-a-program/codesnippet/CSharp/main-and-command-line-arguments_1.cs)]
+
+## Overview
+
+- The `Main` method is the entry point of an .exe program; it is where the program control starts and ends.
+- `Main` is declared inside a class or struct. `Main` must be [static](../../../csharp/language-reference/keywords/static.md) and it should not be [public](../../../csharp/language-reference/keywords/public.md). (In the earlier example, it receives the default access of [private](../../../csharp/language-reference/keywords/private.md).) The enclosing class or struct is not required to be static.
```

Comment:
Fixed.

Note: I looked through old versions of the C# Spec, and this was never true, and never part of the spec. I wonder how this snuck through in this content (because it has been there for quite a while)